dev/release/release-build.sh

```diff
@@ -61,6 +61,9 @@ release-build.sh --release-prepare --releaseVersion="0.11.0-incubating" --develo
 release-build.sh --release-prepare --releaseVersion="0.11.0-incubating" --developmentVersion="0.12.0-SNAPSHOT" --releaseRc="rc1" --tag="v0.11.0-incubating-rc1"
 release-build.sh --release-prepare --releaseVersion="0.11.0-incubating" --developmentVersion="0.12.0-SNAPSHOT" --releaseRc="rc1" --tag="v0.11.0-incubating-rc1"  --gitCommitHash="a874b73" --dryRun
 
+# Create 0.12 RC2 builds from branch-0.12 
```

Comment:
I would add to the explanation of `--gitCommitHash` in the _OPTIONS_ passage above
from:
`--gitCommitHash - Release tag or commit to build from, default master HEAD`
to:
`--gitCommitHash - Release tag, branch name, or commit to build from, default master HEAD`